builder/deploy/uploader.ts

```diff
@@ -43,7 +43,7 @@ export class Uploader {
       console.log("File Error", err);
     });
     const params: PutObjectRequest = {
-      Bucket: options.bucket,
+      Bucket: options.bucket || '',
```

Comment:
Is there a edge case I didn't thought of here? To me it looks like you could upload files to a none existing bucket or at least a bucket without a name?

Reply:
The Bucket can not be undefined, due to which I had to do this.

Reply:
That is true and on line 17 we verify that there is a bucket. So this shouldn't be any issue.